content/docs/command-reference/repro.md

```diff
@@ -83,11 +82,12 @@ $ dvc dag
 ```
 
 This pipeline consists of two parallel branches (`A` and `B`), and the final
-"result" stage, where the branches merge. To reproduce both branches at the same
-time, you could run `dvc repro A2` and `dvc repro B2` at the same time (e.g. in
-separate terminals). After both finish successfully, you can then run
-`dvc repro train`: DVC will know that both branches are already up-to-date and
-only execute the final stage.
+`train` stage, where the branches merge. If you run `dvc repro` at this point,
+it would reproduce each branch sequentially before train. To reproduce both
```

Comment:
You missed the block quotes (hidden but existent in https://github.com/iterative/dvc.org/pull/1624#discussion_r460272991)
```suggestion
it would reproduce each branch sequentially before `train`. To reproduce both
```
I'll just commit this.

Reply:
Thanks!